go.mod

```diff
@@ -0,0 +1,13 @@
+module github.com/lunemec/zapext
```

Comment:
Not sure this is the right module name 🙂

Reply:
Yeah. I hate Go modules. WTF is this shit. Ideas what to do about this except to not commit this?

Reply:
Sure, just use `github.com/tchap/zapext`. This does not mean anything when you are developing the package. You can just clone it from your fork. But if you want to use it under `github.com/tchap/zapext` in your projects for testing, I guess you need to use [replace](https://github.com/golang/go/wiki/Modules#when-should-i-use-the-replace-directive)?